### DISCUSSION TITLE
Upgrade 0.5.2 to 0.6.2, existing streams no longer visible

### DISCUSSION THREAD

**Discussion by johnp789** [openobserve/openobserve#1563] [Q&A] [answered]
Hi, 

I have Openobserve running in Docker, using a mapped volume for storage.  In 0.5.2, the UI shows that I have 2 streams with a total of 16.18 GB ingested.  After updating to 0.6.2, the UI shows only 1 stream, and all my old logs are gone.  If I downgrade to 0.5.2, my old logs are visible again.  Some release notes for 0.6.x versions mentioned letting the team know if this happens.  I'm happy to look around and gather some information from my setup if it would help.  This is a "home lab" setup and is OK if I take it offline, lose data, etc.

Cheers, 
John

**Comment by hengfeiyang** [comment]
First it won't lost data. Can you share your docker start script?

**Reply by johnp789** [reply]
That's right, my data is all there and visible after I switch back to 0.5.2.  I use HashiCorp Nomad to manage the Openobserve service.  Here is my Nomad job definition.

```
job "openobserve" {
  datacenters = ["dc1"]
  type        = "service"
  group "openobserve" {
    count = 1
    network {
      port "http" {
        to     = "5080"
      }
    }
    service {
      provider = "nomad"
      port     = "http"
      check {
        type     = "http"
        path     = "/"
        interval = "10s"
        timeout  = "2s"
      }
    }
    volume "openobserve" {
      type = "host"
      read_only = false
      source = "openobserve"
    }
    task "openobserve" {
      driver = "docker"
      config {
        image = "public.ecr.aws/zinclabs/openobserve:v0.5.2"
        # image = "public.ecr.aws/zinclabs/openobserve:v0.6.2"
        ports = ["http"]
      }
      volume_mount {
        volume      = "openobserve"
        destination = "/data"
      }
      resources {
        cpu    = 500 # MHz
        memory = 2048 # MB
      }
      template {
        # email and password are redacted
        data        = <<-EOH
          ZO_DATA_DIR="/data"
          ZO_ROOT_USER_EMAIL=me@example.com
          ZO_ROOT_USER_PASSWORD=abcdef
          EOH
        destination = "secrets/file.env"
        env         = true
      }
    }
  }
}
```

**Reply by hengfeiyang** [reply, marked as answer]
This looks like no problem, can you try to delete the file `/data/db/metadata.sqlite` and upgrade to `0.6.0` first?

**Reply by hengfeiyang** [reply]
Yes, you need to upgrade to `0.6.0` first, it will run auto-migration to `0.6.x` data structure, then you can upgrade to `0.6.2`.

**Reply by johnp789** [reply]
Thanks!  I stopped the `0.5.2` job, deleted the `metadata.sqlite` file manually, then started `0.6.0` and finally upgraded to `0.6.2`.  At the end, I have all my old streams visible in `0.6.2`.
